packages/mesh-graphql-client/src/types.ts

```diff
@@ -339,8 +339,8 @@ export interface StringifiedRejectedOrderResult<K extends GenericStringifiedSign
 
 export interface StringifiedOrderEvent {
     timestamp: string;
-    order?: StringifiedOrderWithMetadata;
-    orderv4?: StringifiedOrderWithMetadataV4;
+    order?: StringifiedOrderWithMetadata | null;
+    orderv4?: StringifiedOrderWithMetadataV4 | null;
```

Comment:
Do we need `null` here? The `?` already implies a `| undefined`, afaik.

Reply:
Updated to only use `null` as that is what graphql will return 👍